README.md

```diff
@@ -0,0 +1,136 @@
+# Welcome to CitrineOS
+
+CitrineOS is an open-source project aimed at providing a modular server runtime for managing Electric Vehicle (EV) charging infrastructure. This README will guide you through the process of installing and running CitrineOS.
+
+This is the main part of CitrineOS containing the actual charging station management logic, OCPP message routing and all modules.
+
+All other documentation and the issue tracking can be found in our main repository here: https://github.com/citrineos/citrineos.
+
+## Overview
+
+CitrineOS is developed in TypeScript and runs on `NodeJS` with [ws](https://github.com/websockets/ws) and [fastify](https://fastify.dev/).
+
+The system features:
+
+- Dynamic OCPP 2.0.1 message schema validation, prior to transmission using `AJV`
+- Generated OpenAPIv3 specification for easy developer access
+- Configurable logical modules with decorators
+  - `@AsHandler` to handle incoming OCPP 2.0.1 messages
+  - `@AsMessageEndpoint` to expose functions allowing to send messages to charging stations
+  - `@AsDataEndpoint` to expose CRUD access to entities defined in `10_Data`
+- Utilities to connect and extend various message broker and cache mechanisms
+  - Currently supported brokers are `RabbitMQ` and Google Cloud `PubSub`
+  - Currently supported caches are `In Memory` and `Redis`
+
+For more information on the project go to [citrineos.github.io](https://citrineos.github.io).
+
+## Getting Started
+
+### Prerequisites
+
+Before you begin, make sure you have the following installed on your system:
+
+- Node.js (v18 or higher): [Download Node.js](https://nodejs.org/)
+- npm (Node Package Manager): [Download npm](https://www.npmjs.com/get-npm)
+
+### Installation
+
+1. Clone the CitrineOS repository to your local machine:
+
+    ```shell
+    git clone https://github.com/citrineos/citrineos-core
+    ```
+
+1. Navigate to the CitrineOS Server directory:
+
+    ```shell
+    cd citrineos-core/50_Server
+    ```
+
+1. Install project dependencies:
+
+   ```shell
+   ./unix-init-install-all.sh
+   ```
+
+1. Start the server and its supporting infrastructure with:
+
+    ```shell
+    docker-compose up -d 
+    ```
+
+### Configuration
+
+CitrineOS requires configuration to allow your OCPP 2.0.1 compliant charging stations to connect. 
+
+We recommend running and developing the project with docker-compose set-up.
+
+Starting the Server without Docker configuration is also possible. To change necessary configuration for local execution, please adjust the configuration file at `50_Server/src/config/envs/local.ts`. Make sure any changes to the local configuration do not make it into your PR.
+
+### Starting the Server
```

Comment:
We don't want someone to jump to this section of the README and think this is the preferred way to start the server (that would be docker-compose up)
Please make this "Starting the Server without Docker configuration"